Added missing text for pod (#36437)
kubernetes/website: content/en/docs/reference/glossary/volume.md

- core-object
- fundamental
---
 A directory containing data, accessible to the {{< glossary_tooltip text="containers" term_id="container" >}} in a {{< glossary_tooltip text="pod" term_id="pod" >}}.

<!--more-->

> Kubernetes supports many types of volumes. Ephemeral volumes have a lifetime of a pod whereas Persistent volumes exist beyond the lifetime of a pod though for any kind of volume in a given pod, data is always preserved across container restarts.

Reviewer: WDYT?
```suggestion
Kubernetes supports two fundamental types of volumes - ephemeral and persistent. 
Ephemeral volumes exist for the lifetime of their associated pods, while persistent 
volumes are designed to outlive the pods reading from them. However, irrespective
of the volume type, data is always preserved across container restarts.
```

Author: I think the changes makes it more clear and defined though i was wondering that at line 20, 
**persistent volumes are designed to outlive the  pods `reading` from  them.**
Please correct me if i'm wrong , Volumes are designed such that pods can perform the `CRUD` on it.
So to avoid the hassle we can simply go with `Persistent volumes are designed to exist beyond the lifetime of pods associated to it.`

Reviewer: Good catch, thank you! Persistent volumes are designed to exist beyond the lifetime of pods associated with it sounds good, yes.

Author: Okay I'll make the commit with suggested changes.
Thank you for feedback and suggested changes @divya-mohan0209 .

- core-object
- fundamental
---
 A directory containing data, accessible to the {{< glossary_tooltip text="containers" term_id="container" >}} in a {{< glossary_tooltip text="pod" term_id="pod" >}}.

<!--more-->

> Kubernetes supports two fundamental types of volumes - Ephemeral and Persistent. 

Reviewer: I don't think we need capitalization here.
```suggestion
Kubernetes supports two fundamental types of volumes - ephemeral and persistent. 
```

> Ephemeral volumes exist for the lifetime of their associated pods, while Persistent volumes are designed to exist beyond the lifetime of pods associated with it. However, irrespective of the volume type, data is always preserved across container restarts.

Reviewer: ```suggestion
Ephemeral volumes exist for the lifetime of their associated pods, while persistent volumes are designed to exist beyond the lifetime of pods associated with them. However, irrespective of the volume type, data is always preserved across container restarts.
```

- core-object
- fundamental
---

 A directory containing data, accessible to the {{< glossary_tooltip text="containers" term_id="container" >}} in a {{< glossary_tooltip text="pod" term_id="pod" >}}.

<!--more-->

Kubernetes supports two fundamental types of volumes - ephemeral and persistent. 

> Ephemeral volumes exist for the lifetime of their associated pods, while persistent volumes are designed to exist beyond the lifetime of pods associated with them. However, irrespective of the volume type, data is always preserved across container restarts.

Reviewer 2: I prefer the original. The original is concise and clear.

- core-object
- fundamental
---

>  A directory containing data, accessible to the {{< glossary_tooltip text="containers" term_id="container" >}} in a {{< glossary_tooltip text="pod" term_id="pod" >}}.

Reviewer 3: The existing page is OK, I think.

To me I'm afraid this feels like a [trivial edit](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits).

If you'd like to make a bigger revision to the glossary entry for _volume_, we'd welcome that @i-dipanshu. I can see that the content is a bit out of date.

Author: Thank You,  @sftim for the feedback.
I will update the content glossary entry for volume and more over I'll look into all the glossary entries which are out of date and update them as well.
